openshift/cm.yml

```diff
@@ -67,6 +67,12 @@ parameters:
 - name: READABLE_OPENDEVSTACK_REPOS
   required: true
   description: Repositories which the project specific technical user can read
+- name: WEBHOOK_PROXY_EVENTS
+  required: true
+  description: list of supported webhook events
+- name: PROV_APP_WEBHOOKPROXY_HOST
```

Comment:
I don't find this parameter in https://github.com/opendevstack/ods-core/blob/master/configuration-sample/ods-core.env.sample?